src/components/FormNavigation/FormNavigation.tsx

```diff
@@ -47,6 +48,13 @@ const FormNavigation: React.FC<FormNavigationProps> = ({
     {!hideAbortButton && (
       <AbortButton isAuthenticated={isAuthenticated} onDestroySession={onDestroySession} />
     )}
+
+    <Link to="#main-content" className="openforms-backtotop-link">
```

Comment:
the `to` value should be passed in as an option, because the SDK itself does not render this anywhere, it's managed by the backend: https://github.com/open-formulieren/open-forms/blob/49fdc74dbcf14601f6f2158cef67d79e2d18acc6/src/openforms/templates/master.html#L34

This will be broken for customers that embed the SDK in their own CMS (see https://open-forms.readthedocs.io/en/stable/developers/embedding.html for docs on embedding)

Reply:
I've added `backToTopRef` to store id of the target node